src/GLM.jl

```diff
@@ -54,14 +55,14 @@ module GLM
         updateμ!,      # update the response type from the linear predictor
         wrkresp         # working response
 
-    typealias FP AbstractFloat
-    typealias FPVector{T<:FP} AbstractArray{T,1}
+    const FP = AbstractFloat
+    @compat const FPVector{T<:FP} = AbstractArray{T,1}
```

Comment:
Technically I think `const` isn't needed, but that's so subtle that I don't think we care.